src/agent/src/config.rs

```diff
@@ -91,6 +93,7 @@ pub struct AgentConfigBuilder {
     pub unified_cgroup_hierarchy: Option<bool>,
     pub tracing: Option<bool>,
     pub endpoints: Option<EndpointsConfig>,
+    pub use_external_download_helper: Option<bool>,
```

Comment:
I'd prefer if we don't have any setting at all and decide if we're going to use skopeo or not by checking if `SKOPEO_PATH` is installed or not. If it is installed we use it to download images. If it's not we're falling back to the rust implementation.

Reply:
Yes, it will be much simpler for test and runtime selection, I switch to this one